pkg/api/server.go

```diff
@@ -245,6 +257,14 @@ func (s *Server) Start() error {
 
 	log.Infof("Starting server - binding to %s", apiBind)
 
+	var err error
```

Comment:
didn't we say the URL for the API calls should be configurable? With ours as default value.

Reply:
Good catch. This value is now configurable, and has a default equal to the value that was previously hardcoded.